src/test/java/teammates/test/cases/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -449,6 +451,76 @@ private void testSubmitAction() throws Exception {
                                                    "SFSubmitUiT.charlie.d@gmail.tmt"));
     }
 
+    private void testCommentsAction() throws IOException {
+        ______TS("add comments on MCQ questions");
+
+        testCommentsOnMcqQuestions();
+    }
+
+    private void testCommentsOnMcqQuestions() throws IOException {
+        ______TS("add comment on question without response: no effect");
+
+        logout();
+        submitPage = loginToStudentFeedbackSubmitPage("Charlie", "Open Session");
+        submitPage.waitForPageToLoad();
+        submitPage.addFeedbackResponseComment("showResponseCommentAddForm-0-1-6", "Comment without response");
+        submitPage.addFeedbackResponseComment("showResponseCommentAddForm-1-1-7", "Comment without response");
+        submitPage.addFeedbackResponseComment("showResponseCommentAddForm-0-1-10", "Comment without response");
+        submitPage.addFeedbackResponseComment("showResponseCommentAddForm-0-1-12", "Comment without response");
+        submitPage.addFeedbackResponseComment("showResponseCommentAddForm-0-1-16", "Comment without response");
```

Comment:
Could we add "showResponseCommentAddForm" as part of the `addFeedbackResponseComment()` method, so that this part of the string will not have to be passed in as a parameter?